charts/kubernikus-system/vendor/eventexporter/templates/deployment.yaml

```diff
@@ -24,14 +24,17 @@ spec:
         image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
         imagePullPolicy: {{ .Values.image.pullPolicy }}
         args:
+        - -discard=60
```

Comment:
Can we make this a golang duration field? So that you can specify `60s` `5m` etc.

Reply:
Sure.

Reply:
@databus23 I changed this